src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services31_invalid_tags.yml

```diff
@@ -0,0 +1,7 @@
+services:
+    _defaults:
+        tags: ['foo']
+
+    Foo\Bar:
+        tags: invalid
+        inherit_tags: true
```

Comment:
But this way (using a string instead of an array) to specify tags already wasn't supported before. Why should we do that now?